src/View/mainpage/DicomView.py

```diff
@@ -70,21 +70,24 @@ def init_view(self):
         self.view.setBackgroundBrush(background_brush)
     
     def wheelEvent(self, event: QtGui.QWheelEvent):
+        degree = 8
+        step = 15
+        #https://doc.qt.io/qt-6/qwheelevent.html#angleDelta
```

Comment:
Great.  This gets the job done of clarifying where the values come from, and turning them in to variables.